profiles/local-frontend-and-api.js

```diff
@@ -10,6 +10,7 @@ routes[`/beta/${SECTION}/${APP_ID}`] = { host: `http://localhost:${FRONTEND_PORT
 routes[`/${SECTION}/${APP_ID}`]      = { host: `http://localhost:${FRONTEND_PORT}` };
 routes[`/beta/apps/${APP_ID}`]       = { host: `http://localhost:${FRONTEND_PORT}` };
 routes[`/apps/${APP_ID}`]            = { host: `http://localhost:${FRONTEND_PORT}` };
+routes[`/api/system_baseline`]       = { host: `http://localhost:8085` };
```

Comment:
can you change `const API_PORT = 8080;` to be `DRIFT_PORT` and `BASELINE_PORT`, then use `BASELINE_PORT` here?